### PR TITLE
SIG-2114 Documentation Refactor API Key to Usertoken

### PR DESCRIPTION
**Pull request by Nuss9**
Document refactor API Key to Usertoken.

### CODE REVIEW

**Comment by Nuss9** [comment]
Is it correct the API headers need to be adjusted too so that it accepts an Usertoken instead of an API Key?

File: _data/endpoint_examples.yaml [hunk 1]
```diff
@@ -2,7 +2,7 @@ get/api/transaction/{transactionId}:
   - example:
     curl_request: |
       curl \
-        -H "Authorization: APIKey 0123456789abcdef" \
+        -H "Authorization: APIKey <usertokenhere> 0123456789abcdef" \
```
Comment (MrJoe) [inline _data/endpoint_examples.yaml]: Should be:
-H "Authorization: APIKey <usertokenhere>"

File: index.md [hunk 2]
```diff
@@ -27,8 +27,10 @@ We require that all requests are done over SSL.
 To authenticate you have to add two HTTP headers to every request you make.
 The first header includes the name "Application" with the value "APPKey [the app key]".
 This application key will be provided to you by email.
-The second header includes the name "Authorization" with the value "APIKey [your api key]".
-The API key can be generated via My Account > Profile > API keys at the [SignHost Portal](https://portal.signhost.com/).
+The second header includes the name "Authorization" with the value "APIKey [your usertoken]".
+The Usertoken can be generated via My Account > Settings > Usertokens at the [SignHost Portal](https://portal.signhost.com/).
+A name for the usertoken has to be submitted, after which the value for this usertoken will be displayed once only.
+There is no limit on the amount of usertokens that can be generated or deleted.
```
Comment (MrJoe) [inline index.md]: We'll probably limit the amount in though. I think we have a setting to limit it to 64 by default, if more are required this should be requested.

**Comment by Nuss9** [comment]
Applied changes.

**Comment by MrJoe** [comment]
Shouldn’t the same been done for the `APPKey yourclient fedcba0123456789` as well to make it concise?

**Comment by Nuss9** [comment]
Per request, also adapted the App key placeholder to be in the same style as the Usertoken placeholder.

**Pull request merged**